common-lib/common/panels/stat/percentage.libsonnet

```diff
@@ -0,0 +1,16 @@
+local g = import '../../g.libsonnet';
+local base = import './base.libsonnet';
+local stat = g.panel.stat;
+// Panels to display metrics that can go from 0 to 100%.
+// (cpu utilization, memory utilization etc).
+base {
+  new(title, targets, description=''):
+    super.new(title, targets, description)
+    + stat.standardOptions.withDecimals(1)
+    + stat.standardOptions.withUnit('percent')
```

Comment:
nit; Would probably want a parameter to allow for either percent, or percentunit.

Reply:
I'll keep it for now hardcoded and implement it later. For now,  you can always multiply metric by 100.